src/script/conversation/EventInfoEntity.ts

```diff
@@ -62,8 +62,8 @@ export class EventInfoEntity {
     return this.type || (this.genericMessage && this.genericMessage.content);
   }
 
-  setTimestamp(time: string): void {
-    this.timestamp = new Date(time).getTime();
+  setTimestamp(time: string | number): void {
```

Comment:
We should only allow strings or numbers here. If something else arrives, we need to solve the problem at the root.

Reply:
The problem is that the time arrives from the IndexDB where it is saved as a string: `time: "2021-04-15T11:02:34.996Z"`

Reply:
Ah, I figured it out. The time is provided as ISO date string, the tests were just wrong.